com.unity.ml-agents/Runtime/Inference/ModelRunner.cs

```diff
@@ -169,31 +171,33 @@ public void DecideBatch()
             }
 
             Profiler.BeginSample("ModelRunner.DecideAction");
+            Profiler.BeginSample(m_ModelName);
 
-            Profiler.BeginSample($"MLAgents.{m_Model.name}.GenerateTensors");
```

Comment:
These were allocating for the string concatenation, and `m_Model.name` also allocates, so cache it at the start.